search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -242,11 +245,24 @@
     {:name (or archive-center umm-spec-util/not-provided)
      :subOrganizationOf hierarchy}))
 
+(defn check_for_modis_aster
```

Comment:
Is this going to be such a unique case that we won't need to generalize this to a multi-method?

Reply:
Also please rename to use dashes to be inline with code conventions. Suggestion **modis-aster?**

Reply:
I think the long term solution is to move this logic to cmr tags and not drive it through business logic. At least that is what Dana has suggested. Until then I think it is unique.

Reply:
@jaybarra - do you have issue with this still?